scompose/project/instance.py

```diff
@@ -396,11 +397,38 @@ def get_address(self):
         return ip_address
 
     # Logs
+    def set_log_folder(self, params):
+        """set instance's log folder
+           if not user specified, set default folder based on user, home, and host
+
+           Parameters
+           ==========
+           params: dictionary of key, value parameters
+        """
+        self.log_folder = params.get("logs")
+
+        if not self.log_folder:
+            home = get_userhome()
+            user = os.path.basename(home)
+
+            if self.sudo:
+                home = "/root"
+                user = "root"
+
+            # Hostname
+            hostname = platform.node()
+
+            self.log_folder = os.path.join(home, ".singularity", "instances", "logs", hostname, user)
+
+    def get_log_folder(self):
+        """get log folder
```

Comment:
Is there any reason you changed this to a public client function? I think it's more appropriate as private, since it's a helper to the various log functions.